sql/moz-fx-data-shared-prod/search_derived/search_revenue_levers_daily_v1/query.sql

```diff
@@ -1,276 +1,75 @@
-WITH
--- Google Desktop (search + DAU)
-desktop_data_google AS (
+WITH combined_search_data AS (
   SELECT
     submission_date,
-    IF(LOWER(channel) LIKE '%esr%', 'ESR', 'personal') AS channel,
     country,
-    COUNT(DISTINCT client_id) AS dau,
-    COUNT(
-      DISTINCT IF(default_search_engine LIKE '%google%', client_id, NULL)
-    ) AS dau_w_engine_as_default,
-    COUNT(
-      DISTINCT IF(
-        sap > 0
-        AND normalized_engine = 'Google'
-        AND default_search_engine LIKE '%google%',
-        client_id,
-        NULL
-      )
-    ) AS dau_engaged_w_sap,
-    SUM(IF(normalized_engine = 'Google', sap, 0)) AS sap,
-    SUM(IF(normalized_engine = 'Google', tagged_sap, 0)) AS tagged_sap,
-    SUM(IF(normalized_engine = 'Google', tagged_follow_on, 0)) AS tagged_follow_on,
-    SUM(IF(normalized_engine = 'Google', search_with_ads, 0)) AS search_with_ads,
-    SUM(IF(normalized_engine = 'Google', ad_click, 0)) AS ad_click,
-    SUM(IF(normalized_engine = 'Google', organic, 0)) AS organic,
-    SUM(IF(normalized_engine = 'Google', ad_click_organic, 0)) AS ad_click_organic,
-    SUM(IF(normalized_engine = 'Google', search_with_ads_organic, 0)) AS search_with_ads_organic,
-    SUM(IF(normalized_engine = 'Google' AND is_sap_monetizable, sap, 0)) AS monetizable_sap
+    normalized_engine AS partner,
+    'desktop' AS device,
+    sap,
+    tagged_sap,
+    tagged_follow_on,
+    search_with_ads,
+    ad_click,
+    organic,
+    ad_click_organic,
+    search_with_ads_organic,
+    monetizable_sap
```

Comment:
There isn't a `monetizable_sap` field in `moz-fx-data-shared-prod.search.search_aggregates`

Reply:
We've added it to search_aggregates with our latest backfill (which I'm reviewing now!)